KnownIssues.md

```diff
@@ -1,63 +1,31 @@
 
 **ACTION**:
-   Ignore the following messages logged during discover of a 12.2.1 domain.
-
-   ####<Jan 14, 2019 1:14:21 PM> <SEVERE> <CommandExceptionHandler> <handleException> <> <Error: cd() failed.>
-   ####<Jan 14, 2019 1:14:21 PM> <SEVERE> <CommandExceptionHandler> <handleException> <> <Error: ls() failed.>
+Ignore the following messages logged during discover of a 12.2.1 domain.
+```
+<Jan 14, 2019 1:14:21 PM> <SEVERE> <CommandExceptionHandler> <handleException> <> <Error: cd() failed.>
+<Jan 14, 2019 1:14:21 PM> <SEVERE> <CommandExceptionHandler> <handleException> <> <Error: ls() failed.>
+```
 
+### Create Domain Tool with 11G JRF Domains
 
 **ISSUE**:
-   The createDomain tool cannot run rcu for 11g JRF domains. The tool will issue error messages in the log, and
-   terminate the create process.
+The Create Domain Tool cannot initialize RCU for 11g JRF domains. The tool will issue error messages in the log, and terminate the create process.
 
 **ACTION**:
-   Run RCU before executing createDomain for JRF domains
-
-**ISSUE**:
-   The discoverDomain or createDomain tool cannot handle object name with slashes and finished with  
-   warnings.  For example:
-
-    2. WLSDPLY-06140: Unable to cd to the expected path /SelfTuning/NO_NAME_0/WorkManager/wm/SOAWorkManager
-    constructed from location context model_folders = ['SelfTuning', 'WorkManager'],  'name_tokens' =
-    {'SELFTUNING': 'NO_NAME_0','DOMAIN': 'basesoa','WORKMANAGER': 'wm/SOAWorkManager'}; the current folder and
-    its sub-folders cannot be discovered : wlst.cd(/SelfTuning/NO_NAME_0/WorkManager/wm/SOAWorkManager) in
-    offline mode failed: com.oracle.cie.domain.script.ScriptException: No such element WorkManager named wm
+Run the WLS `rcu` command before executing createDomain for JRF domains
 
-   The reason for this warning is because of the slash(es) in the object name. In this case the object name is
-   wm/SOAWorkManager
+### Credential in Security Configuration
 
-**ACTION**:
-   Contact Oracle Support to obtain the patch for the bug number 25790276 for your weblogic version
-   before running the tool.
+**ISSUE**: For WLS versions before 14.1.1, there is a problem setting the `CredentialEncrypted` attribute in the `topology/SecurityConfiguration` folder. The value is not encrypted properly in the configuration, and the domain will fail to start with the error:
+```
+java.lang.IllegalArgumentException: In production mode, it's not allowed to set a clear text value to the property: CredentialEncrypted of SecurityConfigurationMBean
+```
+**ACTION**: Contact Oracle Support to obtain the patch for the bug number 30874677 for your WebLogic version before running the tool.
```

Comment:
for the bug number -> for bug number
your WebLogic version -> your WebLogic Server version